charts/mailpit/templates/ingress.yaml

```diff
@@ -5,8 +5,19 @@ metadata:
   name: {{ template "common.names.fullname" . }}
   namespace: {{ include "common.names.namespace" . | quote }}
   labels: {{- include "common.labels.standard" ( dict "customLabels" .Values.commonLabels "context" $ ) | nindent 4 }}
+  annotations:
+  {{- if .Values.ingress.gce.enabled }}
+    kubernetes.io/ingress.class: gce
+    kubernetes.io/ingress.allow-http: "true"
+    kubernetes.io/ssl-redirect: "true"
+    {{- if .Values.ingress.gce.gcp_managed_certificate.enabled }}
+    networking.gke.io/managed-certificates: {{ template "common.names.fullname" . }}
+    {{- end }}
+  {{- else }}
```

Comment:
let's leave the else empty or use `ingressClassName`

Reply:
Yes, I agree, it should not be set by default! I have removed "else" from this section!